### PR TITLE
More menu adjustments

### PR DESCRIPTION
**Pull request by ceski-1**
I'm working on gamepad documentation ([example](https://ceski-1.github.io/response-curves/)) and want to update the menus a bit for consistency.

### CODE REVIEW

**Comment by ceski-1** [comment]
<details><summary>screenshots</summary>

woof0009

woof0010

woof0011

</details>

File: src/mn_setup.c [hunk 1]
```diff
@@ -2836,12 +2825,24 @@ static setup_menu_t gen_settings3[] = {
     {"Acceleration", S_THERMO, CNTR_X, M_THRM_SPC, {"mouse_acceleration"},
      .strings_id = str_mouse_accel, .action = G_UpdateMouseVariables},
 
+    MI_GAP,
+
+    // [FG] double click to "use"
+    {"Double-Click to \"Use\"", S_ONOFF, CNTR_X, M_SPC, {"dclick_use"}},
+
+    {"Free Look", S_ONOFF, CNTR_X, M_SPC, {"mouselook"},
```
Comment (rfomin) [inline src/mn_setup.c:2833]: Just out of curiosity, why did you decide to move ‘Free Look’ to the bottom? I suspect that many users will be looking for it in the menu.

**Review by rfomin — APPROVED** [review]
LGTM

File: src/mn_setup.c (hunk 1 above)
Comment (ceski-1) [reply src/mn_setup.c:2833]: I think turn sensitivity is the most important setting for both mouse and gamepad, so it should be the first thing a player sees when moving through the tabs. Free look is less important by comparison. In my opinion, y-shearing is not great to use and players looking for true free look aren't using Woof anyway.

Comment (ceski-1) [reply src/mn_setup.c:2833]: That said, I'm okay with moving it back if it seems too disruptive.

Comment (rfomin) [reply src/mn_setup.c:2833]: > That said, I'm okay with moving it back if it seems too disruptive.

No, looks good as it is.

**Pull request merged**